config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  get '/', to: 'tasks#index', as: 'home'
+
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/tasks', to: 'tasks#index', as: 'tasks'
+  get '/tasks/new', to: 'tasks#new', as: 'new_task'
+  post '/tasks', to: 'tasks#create'
+  get '/tasks/:id', to: 'tasks#show', as: 'task'
+  get '/tasks/:id/edit', to: 'tasks#edit', as: 'edit_task'
+  patch '/tasks/:id', to: 'tasks#update', as: 'update_task'
```

Comment:
This route does not need its own name, because the name for the show route (`task`) would work instead.

This is because the path portion of both routes is exactly the same (`'/tasks/:id'`).